sdks/java/io/hadoop-input-format/src/test/resources/kubernetes/cassandra/data-load.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Identify the pod
+cassandra_pods="kubectl get pods -l name=cassandra1"
+running_seed=$($cassandra_pods --no-headers | \
```

Comment:
I was curious as to whether or not https://kubernetes.io/docs/user-guide/kubectl/kubectl_get/  can help you out with getting the value you want without using awk.  It looks like the custom columns or go template *could* help but I'm not entirely sure it's easier than this (ie, they would involve having another completely separate file) - I was looking at http://blog.kubernetes.io/2015/10/some-things-you-didnt-know-about-kubectl_28.html

The awk generally makes me feel a bit nervous/this feels a bit sensitive.